vite.config.router.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path';
+import { defineConfig } from 'vite';
+import Vue from '@vitejs/plugin-vue';
+import ViteSvgLoader from 'vite-svg-loader';
+import Visualizer from 'rollup-plugin-visualizer';
+
+export default defineConfig({
+  root: '.',
+  resolve: {
+    alias: {
+      '@': path.join(__dirname, 'src'),
+      '@@': __dirname,
+    },
+  },
+  plugins: [
+    Vue(),
+    ViteSvgLoader(),
+    Visualizer({
+      filename: path.resolve(__dirname, 'report/treemap.html'),
+      title: 'Bundle Visualizer',
+      gzipSize: true,
+      brotliSize: true,
+      template: 'treemap',
+    }),
```

Comment:
This will override the one generated for the root lib. You should rename the file